File: JPADemo/src/main/java/com/example/springbootjpa/JPADemo/controllers/TestController.java
package com.example.springbootjpa.JPADemo.controllers;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class TestController {


    @GetMapping("/test")
    public String test() {
        return "Your Spring Boot App is up and running!";
    }
}

File: RestDemo/src/main/java/com/springboot/restDemo/controller/TestController.kt
package com.springboot.restDemo.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/test")
class TestController {


    @GetMapping("/say-hello")
    fun test(): String {
        return "Hello world from Kotlin!!"
    }


}